src/Stratis.Bitcoin.Networks/StratisMain.cs

```diff
@@ -84,7 +85,19 @@ public StratisMain()
 
             var bip9Deployments = new StratisBIP9Deployments()
             {
-                [StratisBIP9Deployments.ColdStaking] = new BIP9DeploymentsParameters(2,
+                [StratisBIP9Deployments.TestDummy] = new BIP9DeploymentsParameters(28,
```

Comment:
Included this for consistency; it isn't really necessary & the activation date is currently arbitrary.